client/backwards_compatibility_test.go

```diff
@@ -80,7 +80,7 @@ func Test0Dot1Migration(t *testing.T) {
 	require.NoError(t, err)
 	require.NoError(t, recursiveCopy("../fixtures/compatibility/notary0.1", tmpDir))
 
-	gun := "docker.com/notary0.1/samplerepo"
+	gun := data.GUN("docker.com/notary0.1/samplerepo")
```

Comment:
Rather than casting in cases like this, it's slightly more idiomatic to declare the type on the variable and go will deal with ensuring the literal has the correct type:

```
var gun data.GUN = "docker.io/notary0.1/samplerepo"
```